### DISCUSSION TITLE
Using folders on mobile

### DISCUSSION THREAD

**Discussion by filiptrplan** [open-webui/open-webui#6672] [Q&A]
How do I move a chat to a folder on mobile?

**Comment by silentoplayz** [comment]
You can press and hold down on a chat within the chat navbar and move it into a folder. It should allow you to drag and drop the chat into a folder. This works for me on my iPhone 13 Pro to move a chat both into and out from a folder.

**Reply by herbras** [reply]
I've tried it on Android, a pop up for Ahref appears

**Reply by filiptrplan** [reply]
Yes, it doesn't work for me either. It brings up the link popup.
